python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -30,7 +30,7 @@
 
 _imported_GangaList = None
 
-do_not_copy = ['_proxyObject', '_proxyClass', '_data', '_index_cache', '_parent', '_registry', '_impl']
+do_not_copy = ['_proxyObject', '_data', '_index_cache', '_parent', '_registry']
```

Comment:
This is used to prevent any copy operation involving ```__dict__``` from going in infinite loops. Is this safe?